launch_testing/launch_testing/__init__.py

```diff
@@ -0,0 +1,74 @@
+import io
+import re
+from launch.output_handler import LineOutput
```

Comment:
The package lacks a run dependency on `lauch`: http://ci.ros2.org/job/ros2_batch_ci_linux/597/testReport/nose.failure/Failure/runTest/

That is why I suggested to run a CI job before merging. Did you run it locally? In isolation?